persistent/ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog for persistent
 
+## Unreleased
+
+* [1497](https://github.com/yesodweb/persistent/pull/1497)
+    * Always generates `SymbolToField "id"` instance
```

Comment:
This will need to be a minor version bump - may be best to roll up with #1503

Reply:
Sounds reasonable to me. Anything I need to do / pay attention to?

Reply:
I'll rebase after #1503 has been merged (adjust the changelog section)